src/main/java/lotto/domain/result/Statistic.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain.result;
+
+import java.util.Arrays;
+
+public enum Statistic {
+	THREE(3, 5000),
+	FOUR(4, 50000),
+	FIVE(5, 1500000),
+	BONUS(5, 3000000),
+	SIX(6, 2000000000);
+
+	private final int matchingNumbers;
+	private final int prize;
+	private int count = 0;
+
+	Statistic(int matchingNumbers, int prize) {
+		this.matchingNumbers = matchingNumbers;
+		this.prize = prize;
+	}
+
+	public static Statistic getRank(int numberOfMatch) throws Exception {
+		return Arrays.stream(values())
+			.filter(statistic -> statistic.matchingNumbers == numberOfMatch)
+			.findFirst()
+			.orElseThrow(Exception::new);
```

Comment:
```suggestion
			.orElseThrow(Exception::new);
```

어떠한 예외처리인지 메시지를 통해 나타내보는 것은 어떨까요?
추가적으로 Exception과 RuntimeException은 어떠한 차이가 있을지 공부해보아요.

Reply:
네. 수정하였습니다!